src/tools/pub/dependencies.md

```diff
@@ -149,40 +133,43 @@ Even if the package repo is private, if you can
 [connect to the repo using SSH,][GitHub SSH]
 then you can depend on the package by using the repo's SSH URL:
 
-{% prettify yaml tag=pre+code %}
+```yaml
 dependencies:
   kittens:
     git: git@github.com:munificent/kittens.git
-{% endprettify %}
+```
 
 If you want to depend on a specific commit, branch, or tag,
-add a `ref` argument:
+add a `ref` key to the description:
 
-{% prettify yaml tag=pre+code %}
+```yaml
 dependencies:
   kittens:
     git:
       url: git@github.com:munificent/kittens.git
       ref: some-branch
-{% endprettify %}
+```
 
 The ref can be anything that Git allows to [identify a commit.][commit]
 
 [commit]: https://www.kernel.org/pub/software/scm/git/docs/user-manual.html#naming-commits
 
-Pub assumes that the package is in the root of the Git repository.
-To specify a different location in the repo, use the `path` argument:
+Pub assumes that the package is in the root of the Git repository. To specify a
+different location in the repo, specify a `path` relative to the repository
+root:
 
-{% prettify yaml tag=pre+code %}
+```yaml
 dependencies:
   kittens:
     git:
       url: git@github.com:munificent/cats.git
       path: path/to/kittens
-{% endprettify %}
+```
 
 The path is relative to the Git repo's root.
 
+Git dependencies are not allowed on [pub.dev][pubsite].
```

Comment:
I wonder if it would be more clear if we specify in what way they are not allowed. What do you think?

```suggestion
Git dependencies are not allowed as dependencies
for packages uploaded to [pub.dev][pubsite].
```

Reply:
Yeah - that's better!